src/ostree/ot-main.c

```diff
@@ -41,6 +41,10 @@ static gboolean opt_verbose;
 static gboolean opt_version;
 static gboolean opt_print_current_dir;
 
+// TODO: make this public?  But no one sane wants to use our C headers
+// to find where to put files.  Maybe we can make it printed by the CLI?
+#define _OSTREE_EXT_DIR PKGLIBEXECDIR "/ext"
```

Comment:
`/usr/share/pkgconfig/ostree.pc` like `/usr/share/pkgconfig/systemd.pc`? Not a blocker, but I think it provides a natural place to provide info about the CLI. Another alternative is to put it in the `--version` output like `Features`.

Reply:
Good suggestion!  Done, thanks!